### PR TITLE
Update translations

### PR DESCRIPTION
**Pull request by lund5000**
* Updates translation files to match new header and footer markup
* Adds package.json for semantic versioning at the request of Cubic/Reflexions

### CODE REVIEW

File: preview/Makefile [hunk 1]
```diff
@@ -4,4 +4,5 @@ serve_preview: copy_files
 copy_files:
 	cp -r ../{fonts,images,favicon.ico,styles.*.css,header.*.js*} .
 	mkdir -p _includes && cp ../{footer.html,header.html} _includes
+	mkdir -p _includes && cp ../languages/{footer-*.html,header-*.html} _includes
```
Comment (thecristen) [inline preview/Makefile:7]: Thank youuu

File: package.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "dotcomchrome",
+  "version": "2.0.0",
+  "description": "Header and footer templates from mbta.com",
```
Comment (thecristen) [inline package.json]:
```suggestion
  "description": "Header and footer templates and assets from mbta.com",
```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mbta/dotcomchrome.git"
```
Comment (thecristen) [inline package.json]:
```suggestion
    "url": "https://github.com/mbta/dotcomchrome.git"
```

I _think_ the proper notation omits the `git+`

File: package.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+  },
+  "homepage": "https://github.com/mbta/dotcomchrome#readme"
```
Comment (thecristen) [inline package.json]: let's add `private: true`

**Review by thecristen — CHANGES_REQUESTED** [review]
Some tiny things, looks reasonable enough!

File: package.json (hunk 4 above)
Comment (lund5000) [reply package.json]: Done!

**Review by thecristen — APPROVED** [review]

**Pull request merged**